### ISSUE TITLE
kube-dns not being created in v0.9.3-rc.4

### ISSUE THREAD

**Issue by iameli** [coreos/kube-aws#247]
Hi,

Just brought up a new cluster and there doesn't seem to be a DNS deployment.

```
> kubectl --kubeconfig kubeconfig get deployments --all-namespaces
No resources found.

> kubectl --kubeconfig kubeconfig get pods --all-namespaces
NAMESPACE     NAME                                                                READY     STATUS    RESTARTS   AGE
kube-system   kube-apiserver-ip-10-25-10-81.us-west-2.compute.internal            1/1       Running   0          6m
kube-system   kube-controller-manager-ip-10-25-10-81.us-west-2.compute.internal   1/1       Running   0          6m
kube-system   kube-proxy-ip-10-25-10-81.us-west-2.compute.internal                1/1       Running   0          6m
kube-system   kube-scheduler-ip-10-25-10-81.us-west-2.compute.internal            1/1       Running   0          5m
kube-system   kubernetes-dashboard-v1.5.1-73k2k                                   0/1       Pending   0          6m
```

It looks like a few things are erroring out in `install-kube-system.service`, probably this is the cause? [This is the full output of journalctl -u install-kube-system](https://gist.github.com/iameli/9ea8b6413e349eb6ed04a12f01ac2b2d), but the relevant bit appears to be...

```
Warning: Couldn't read data from file
Warning: "/srv/kubernetes/manifests//srv/kubernetes/manifests/kube-dns-de.yaml"
Warning: , this makes an empty POST.
```

Looks like that path is getting duplicated somewhere? Very curious.

**Comment by redbaron** [comment]
I am really sorry for that, experienced that myself, here is a fix https://github.com/coreos/kube-aws/pull/245

**Comment by iameli** [comment]
Nice! No worries, will close this when that gets merged

**Comment by whereisaaron** [comment]
Grump grump @redbaron just when I thought I could launch my clusters without patch files!!! 😛 

Hit the same problem, and #245 worked to fix it.

**Comment by mumoshu** [comment]
@iameli @redbaron @whereisaaron Thanks to all for your support!
I've just released v0.9.3-rc.5 with the fix included.
